test/Graphics/Shape.cpp

```diff
@@ -3,6 +3,13 @@
 #include <doctest/doctest.h>
 
 #include <GraphicsUtil.hpp>
+#include <type_traits>
+
+static_assert(!std::is_constructible<sf::Shape>::value);
+static_assert(!std::is_copy_constructible<sf::Shape>::value);
+static_assert(std::is_copy_assignable<sf::Shape>::value);
+static_assert(!std::is_move_constructible<sf::Shape>::value);
+static_assert(std::is_move_assignable<sf::Shape>::value);
```

Comment:
Why are these throwing operations? Ditto for all the other derived shape classes.

Reply:
I'm not certain. At one point I did look into this but forgot what my conclusion was.